TOSegmentedControl/TOSegmentedControl.m

```diff
@@ -613,7 +613,7 @@ - (CGRect)frameForSegmentAtIndex:(NSInteger)index
     CGSize size = self.trackView.frame.size;
     
     CGRect frame = CGRectZero;
-    frame.origin.x = _thumbInset + (self.segmentWidth * index) + ((_thumbInset * 2.0f) * index);
+    frame.origin.x = (self.segmentWidth * index) + ((_thumbInset) * index-2);
```

Comment:
Hmm, what was the cause behind needing to change this value? This seems to not function correctly, and is causing the thumb view to be inset too much when it's all the way to the right hand side.